internal/pkg/backend/local/local.go

```diff
@@ -0,0 +1,47 @@
+package local
+
+import (
+	"io/ioutil"
+	"sync"
+)
+
+const DefaultStateFile = "devstream.state"
+
+// Local is a default implement for backend.Backend
+type Local struct {
+	mu       sync.Mutex
+	filename string
+}
+
+// NewLocal will use DefaultStateFile as statemanager file if filename is not given.
+func NewLocal(filename string) *Local {
```

Comment:
Add two UT to cover this function: empty filename & non-empty filename.

The UT for `backend` or `local` module might be redundant, though, because it's already tested with the `statemanager` module.

Reply:
same as above conversation.